go.work

```diff
@@ -0,0 +1,3 @@
+go 1.25.3
```

Comment:
let me know if this is a good idea. I had to do this locally to get stuff to compile/type check